client/landing/stepper/declarative-flow/sensei.ts

```diff
@@ -53,11 +56,7 @@ const sensei: Flow = {
 				case 'intro':
 					return navigate( 'senseiSetup' );
 				case 'senseiSetup':
-					if ( isLoggedIn ) {
-						return navigate( 'senseiDomain' );
```

Comment:
I marked `senseiDomain` as `private` so as soon the user has access to it they will be automatically redirected to the sign-up flow, no internal logic is required.